homework6/Test/test_Task1.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from Task1 import instances_counter
+
+
+@instances_counter
+class A:
+    def get_one(self) -> int:
+        return 1
+
+
+@instances_counter
+class B:
```

Comment:
Add a docstring to the class and check if the name of the class is the same and the dockstring is there too.